pyTigerGraph/gds/gds.py

```diff
@@ -1469,9 +1469,25 @@ def featurizer(
         self,
         repo: str = None, 
         algo_version: str = None) -> Featurizer:
-        """Get a featurizer.
-            Returns:
-                Featurizer
+        """Get a featurizer. The Featurizer enables installation and execution of algorithms in the Graph Data Science (GDS) libarary. 
+        The Featurizer pulls the most up-to-date version of the algorithm available in our public GitHub repository that is
+        compatible with your database version.
+        Note: In environments not connected to the public internet, you can download the repository manually and use the featurizer
+        like this:
+        ```
+        import pyTigerGraph as tg
+        from pyTigerGraph.gds.featurizer import Featurizer
+
+        conn = tg.TigerGraphConnection(host="HOSTNAME_HERE", username="USERNAME_HERE", password="PASSWORD_HERE", graphname="GRAPHNAME_HERE")
+        conn.getToken(conn.createSecret())
+        feat = Featurizer(conn, repo="PATH/TO/MANUALLY_DOWNLOADED_REPOSITORY")
```

Comment:
why not using conn.gds.featurizer(repo = ...)?

Reply:
Do we want to add that parameter to the factory function? I was thinking it makes the factory a bit more clunky. Can definitely add it though, there wasn't a strong reason why I didn't add it in.

Reply:
looks like it is already in the factory function lol so no need for extra work